app/models/queries/order_by_subscription_query.rb

```diff
@@ -0,0 +1,45 @@
+module Queries
+  class OrderBySubscriptionQuery < UserQuestionsQuery
+    attr_reader :user
+
+    def initialize(user)
+      @user = user
+    end
```

Comment:
I think we can also leverage on the **inheritance** and make user of the `initializer` in the `UserQuestionsQuery`